model_tuning/embedding.py

```diff
@@ -0,0 +1,100 @@
+import os
+import pickle
+import sys
+from typing import List
+
+import torch
+from sentence_transformers import SentenceTransformer
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from utils.parse_and_extract_loinc_names import parse_snoinc_extracts
+
+SNOINC_CODES_FILE = "../data/snoinc_extracts/loinc_lab_names_20251008.csv"
```

Comment:
I'm fine with hard coding this file name here if we have to, but if we do come out with newer versions of the SNOINC extracts we will have to update this here.

Reply:
Yeah for sure. With the embedding notebook, we'll only need to run this prior to actual prod deployments, at least for now. We'll be  making all our embeddings before we ever upload to AWS and then dumping them in a vector DB, so this won't need to get touched until we implement functionality to handle changes in LOINC.